src/server/transaction.h

```diff
@@ -280,10 +312,12 @@ class Transaction {
   };
 
   struct PerShardCache {
+    bool requested_active = false;
```

Comment:
This is what I added to schedule on shards without having the keys.

If this flag is set, the new ACTIVE flag is set as well even if there are no arguments. 

TODO: need to add a comment about this